doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -19,11 +19,17 @@
 
 ## Synopsis
 
-Ensure Reference Implementation (RI) of CNTT Reference Model (RM) and CNTT Reference Architecture (RA) meets industry driven quality assurance standards for compliance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
+Ensure Reference Implementation (RI) of CNTT Reference Model (RM) and CNTT Reference Architecture (RA) meets industry driven quality assurance standards for conformance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
 
-For the purposes of this chapter, NFVI+VNF testing will be performed for **Conformances** indicating a measured performance of the adherence to, and demonstrated proficiency with, all aspects of software delivery including but not limited to planning, logistics for communication, and testing of installed, or implemented, NFVI.  Similarily, VNFs will be checked for **Compliance** and **Validations** against the RM and RA requirements ensuring VNF instantiation, stability, and successful removal of the VNF from the implementation of the stated architecture.  The additional VNF testing will verify that the developed VNF meets prescribed RM/RA infrastructure standards.
+For the purposes of this chapter, NFVI+VNF testing will be performed to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of software delivery.  More specifically, Conformance includes:
 
-In summary, NFVI+VNF testing will be performed for **Verification** and **Validation** purposes.
+ - Verified implementations of NFVI+VNF match expected design requirements
+ - Clearly stated guidelines for test, badging, and lifecycle management
+ - Inclusion of Operational run-books for 3rd party supplier instantiation and validations of NFVI+VNF0
+ - Evidence, through test results (FR/NFR), confirming delivered code matches design and client expectations
+ - Interoperability testing of Reference VNFs ensuring integration stability, including the successful removal of the VNF from the target architecture
```

Comment:
I would rework this to, "Interoperability testing with Reference VNFs, ensuring integration stability and life-cycle management of the Reference VNF on the target architecture."

Reply:
It still seems weird to me that we are talking only about removal of the VNF, plus, why is it "target architecture" instead of NFVI (architecture makes me think about things like aarch64 etc.). Isn't this really saying VNF integration, stability, and life-cycle on the NFVI under test are in scope of the testing?

Reply:
Rather than "target architecture" should it be "target implementation"?  Is it removal of the VNF from the "infrastructure" that it was deployed to, that you're looking to prove?

Reply:
@tomkivlin , agree.  Changed "architecture" for "implementation".  More accurate.  Thank you.

@lylavoie , references to "removal [of vnf]" have been removed.  You are correct, it was not appropriate to put that comment in that paragraph.  

Saved changes if you can take another look please.